src/bridge.ts

```diff
@@ -1,24 +1,41 @@
 import { ipcRenderer as IpcRenderer } from 'electron';
 
+import { IPC_CHANNELS } from '@types';
+
 type Unsubscribe = () => void;
 type Listener = (...args: any[]) => void;
 
 export interface IIpcBridge {
```

Comment:
IMHO we should a lot stricter here and only run an authorised and strongly typed list of methods instead of generics. How about we move the services into the bridge ? This way we have a single place to define ipc communication ?